### DISCUSSION TITLE
[storage/retry-limit-exceeded] Storage uploadTask not uploading file to Firebase Storage

### DISCUSSION THREAD

**Discussion by thaophung137** [firebase/flutterfire#10822] [Q&A]
Hello everyone,

I am working on a chat app where the user can upload their profile image. My plan is to upload the file to firebase storage and save the download url to firestore. I implemented this and it worked for a day... then now everytime I try uploading an image, I get this error after 5 minutes:
The maximum time limit on an operation (upload, download, delete, etc.) has been excceded. Try uploading again. 

This is the function I have for uploading:
<img width="350" alt="upload" src="https://user-images.githubusercontent.com/42352705/233227395-2bd6f4ee-692b-464b-8b25-c3c1c7fc2374.png">

For some reason, it never get pass this line:
<img width="351" alt="Screenshot 2023-04-19 173617" src="https://user-images.githubusercontent.com/42352705/233227959-0197eb29-603e-4a99-ae36-ce3bf66b3d29.png">

<img width="529" alt="Screenshot 2023-04-19 173858" src="https://user-images.githubusercontent.com/42352705/233228310-4be54aaf-e351-4f24-b8b3-b49effaf78cf.png">

**Comment by kodobrody** [comment]
literally having the exact same issue, although using entirely different stack (next.js) also trying to upload just images

**Comment by TsungMin-Chang** [comment]
Same here, upload hangs and then gives timeout error. Couldn't get the Firebase modular APIs to work properly.

**Comment by AnushaKN16** [comment]
Even I am facing the above issue. Any solution?
